tread                : 0.4350
wheel_diameter_right : 0.386
wheel_diameter_left  : 0.385
